### PR TITLE
Restaurant multiple menus

### PR DESCRIPTION
**Pull request by oliverochman**
### PT-link
https://www.pivotaltracker.com/story/show/154208645

### CODE REVIEW

File: features/step_definitions/product_categories_steps.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 Given("the following products exist within a specific restaurant and category") do |table|
   table.hashes.each do |product|
     restaurant = Restaurant.find_by(name: product[:restaurant])
+    menu = FactoryBot.create(:menu, name: product[:menu], restaurant: restaurant)
```
Comment (lcegit) [inline features/step_definitions/product_categories_steps.rb:4]: Possible refractoring for 'FactoryBot' -- it should be sufficient to just write 'create'

**Review by lcegit — COMMENTED** [review]
Looks good!

File: features/step_definitions/product_categories_steps.rb (hunk 1 above)
Comment (tochman) [reply features/step_definitions/product_categories_steps.rb:4]: Good point @lcegit Not a blocker atm though.

**Review by tochman — APPROVED** [review]
Good job. Please revisit some files and fix formatting in next pr's

**Pull request merged**